skills/accelerated-computing-cudf/evals/files/cudf-null-handling/code/null_pipeline.py

```
@@ -0,0 +1,142 @@
+        filled.loc[mask, "humidity"] = median_val
+    # Fill remaining humidity nulls with global median
+    filled["humidity"] = filled["humidity"].fillna(filled["humidity"].median())
+
+    print(f"After fills, remaining nulls:\n{filled.isnull().sum()}")
+    return filled
+
+
+def interpolate_pressure(df):
+    """Interpolate pressure readings within each station."""
+    interp_frames = []
+    for station, group in df.groupby("station_code"):
```

Ruff (0.15.14)

[warning] Loop control variable `station` not used within loop body

Rename unused `station` to `_station`

(B007)